modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/error.jsp

```diff
@@ -21,4 +21,28 @@
 <liferay-ui:error exception="<%= NoSuchOrganizationException.class %>" message="the-organization-could-not-be-found" />
 <liferay-ui:error exception="<%= NoSuchUserException.class %>" message="the-user-could-not-be-found" />
 
-<liferay-ui:error-principal />
\ No newline at end of file
+<liferay-ui:error-principal />
+
```

Comment:
I think this was made by mistake

Reply:
when aligned this branch with master i found out many conflicts, I thought they were modifications already merged. Have i to remove that?